end-to-end-tests/tst/helpers/resource_status.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+
+# contains all the the cloudformation stack resource status
+class ResourceStatus(Enum):
```

Comment:
What's the value of this being an Enum here?

Reply:
To state that the ResourceStatus is a constant so I used Enum